### ISSUE TITLE
I cannot code to use  individual elements of a tensor outside a session 

### ISSUE THREAD

**Issue by fernande2000** [tensorflow/tensorflow#19725]
I am writing a model in Keras, where the output of one of the layers gives me the coordinates of a mask that I want to apply on the image input. The problem is that I could not find any way to do this, methods such as tf.slice() do not work if you are not inside a session. 
I want to do equivalent to this:
x=dense(2)(x)
A=np.zeros((224,224, 3)) or K.zeros((224,224,3))
x=int(x)
A[x[0],x[1],:]=1
where x comes from previous layers and is reassigned in the layer shown (dense layer from keras), the output is two numbers representing the position in the picture, then I create A which is a mask to be multiplied with the input (input not shown), and I need both, make the output of the layer an integer, and then use it to add ones to the mask A at position (x[0],x[1]) (and other ones too, but just having one is enough to know the method.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by drpngx** [comment]
You can't write to individual elements.

Tensorflow (in graph mode) is a declarative framework. You describe the computation first, in python, then use `session.run(..)` to execute the computation in another step.

With eager mode, you can run in python as you normally would. Please try it out.

**Comment by drpngx** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
